pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -908,6 +936,109 @@ func createEcdhCert() error {
 	return nil
 }
 
+// create Ecdh Template
+func createEcdhSoftTemplate() x509.Certificate {
+	serialNumberLimit := new(big.Int).Lsh(big.NewInt(1), 128)
+	serialNumber, _ := rand.Int(rand.Reader, serialNumberLimit)
+	notBefore := time.Now()
+	// twenty years
+	notAfter := notBefore.AddDate(20, 0, 0)
+
+	template := x509.Certificate{
+		SerialNumber: serialNumber,
+		Subject: pkix.Name{
+			Country:      []string{"US"},
+			Province:     []string{"CA"},
+			Locality:     []string{"Santa Clara"},
+			Organization: []string{"Zededa, Inc"},
+			CommonName:   "Cipher Block",
```

Comment:
That common name does not match what we use for ECDH on existing devices. 
I suggest we call both of them the same and have it be "Device ECDH certificate"